### PR TITLE
feat: add lua filter to monitor exporter

### PR DESCRIPTION
**Pull request by bojiang**
Background: The `in_opentelemetry` of fluent bit generates unstructured fluentbit messages. And neither build-in JSON Parser nor `out_bigquery` supports unstructured messages.

In this PR we just parse the unstructured message with a custom lua script.

ref: [rxi](https://github.com/rxi)/[json.lua](https://github.com/rxi/json.lua) (MIT License)

### CODE REVIEW

File: controllers/bentodeployment_controller.go [hunk 1]
```diff
@@ -2255,7 +2255,7 @@ monitoring.options.insecure=true`
 		lastPort++
 		monitorExporterProbePort := lastPort
 
-		monitorExporterImage := "quay.io/bentoml/bentoml-monitor-exporter:0.0.2"
+		monitorExporterImage := "quay.io/bentoml/bentoml-monitor-exporter:0.0.3"
```
Comment (yetone) [inline controllers/bentodeployment_controller.go:2258]: can be an environment var like these?

https://github.com/bentoml/yatai-deployment/blob/main/helm/yatai-deployment/values.yaml#L112

Comment (bojiang) [reply controllers/bentodeployment_controller.go:2258]: https://github.com/bentoml/yatai-deployment/pull/107/commits/c9ce618e990756b5648ca41fe2e105fd0d8903e0

File: container_images/bentoml-monitor-exporter/Makefile [hunk 2]
```diff
@@ -1,5 +1,7 @@
-IMG ?= quay.io/bentoml/bentoml-monitor-exporter:0.0.2
+IMG ?= quay.io/bentoml/bentoml-monitor-exporter:0.0.3
 
 build:
 	docker build . --tag $(IMG)
+
+push:
```
Comment (yetone) [inline container_images/bentoml-monitor-exporter/Makefile:6]: Follow the existing specification or change all Makefiles to the same specification. How about this?

https://github.com/bentoml/yatai-deployment/blob/main/container_images/bento-debugger/Makefile#L3

File: controllers/bentodeployment_controller.go [hunk 3]
```diff
@@ -2255,7 +2255,11 @@ monitoring.options.insecure=true`
...
```
Comment (yetone) [inline controllers/bentodeployment_controller.go:2258]: How about renaming this value to `defaultMonitorExporterImage` and changing `monitorExporterImage_` to `monitorExporterImage`?

File: controllers/bentodeployment_controller.go (hunk 1 above)
Comment (yetone) [reply controllers/bentodeployment_controller.go:2258]: I think you should also add new values to the helm chart values file and modify the helm template yaml like this: https://github.com/bentoml/yatai-deployment/blob/main/helm/yatai-deployment/templates/secret-env.yaml#L14

File: container_images/bentoml-monitor-exporter/Makefile (hunk 2 above)
Comment (bojiang) [reply container_images/bentoml-monitor-exporter/Makefile:6]: sure

**Pull request merged**